File: be/src/test/java/sg/flow/FlowApplicationTests.java
package sg.flow;

import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import sg.flow.FlowApplication;

import static org.junit.jupiter.api.Assertions.assertNotNull;

@SpringBootTest
class FlowApplicationTests {

        @Autowired
        private FlowApplication flowApplication;

        @Test
        void contextLoads() {
                assertNotNull(flowApplication);
        }

}

File: be/src/test/kotlin/sg/flow/FlowApplicationTests.kt
package sg.flow

import org.junit.jupiter.api.Assertions.assertNotNull
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest

@SpringBootTest
@org.springframework.test.context.ActiveProfiles("test")
class FlowApplicationTests {

    @Autowired private lateinit var flowApplication: FlowApplication

    @Test
    fun contextLoads() {
        assertNotNull(flowApplication)
    }
}
